packages/perseus/src/__tests__/renderer.test.tsx

```diff
@@ -201,18 +201,6 @@ describe("renderer", () => {
             expect(renderer.state.lastUsedWidgetId).toBeNull();
 
             expect(renderer.state.widgetInfo).toStrictEqual(question1.widgets);
-            expect(renderer.state.widgetProps).toEqual({
```

Comment:
`renderer.state.widgetProps` is essentially Renderer's stash of RenderProps which this PR removes.